canonical/ubuntu.com
# Rename /pro/beta to pro/tutorial with redirect

## Done

Rename /pro/beta to pro/tutorial with redirect

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- Go to /pro/beta make sure you are redirected to /pro/tutorials


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1471
